### ISSUE TITLE
Could not find a version that matches prompt-toolkit<2.0.0,<2.1.0,>=1.0.0,>=2.0.0

### ISSUE THREAD

**Issue by Someguy123** [steemit/jussi#211]
I'm not sure if this is a problem with the Dockerfile, or with jussi itself. But when building with `docker build -t "jussi" .` in the root folder, pipenv throws an error that it cannot find prompt-toolkit. 

I couldn't see this in `Pipenv`, so it might be a sub dependency. The problem seems to be because the package name is `prompt_toolkit` not `prompt-toolkit`.

**The issue:** When building with docker from the latest master commit, an error `Could not find a version that matches prompt-toolkit<2.0.0,<2.1.0,>=1.0.0,>=2.0.0` is raised, stopping the build

**Reproduction:**

```
git clone https://github.com/steemit/jussi.git
cd jussi
docker build -t jussi .
```

**Running on branch:** `master`

**Current Commit:** `d67c679b8248a3c299d35db131089bfa4eb0048a`

**Docker version:** `18.01.0-ce, build 03596f5`

**Kernel Version:** `Linux 4.4.0-109-generic #132-Ubuntu SMP Tue Jan 9 19:52:39 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux`

**Output of** `docker build -t "jussi" .` (truncated the successful intermediaries above)

```
---> Running in e25232eedd7b
Removing intermediate container e25232eedd7b
 ---> 29f91ec56ed7
Step 23/28 : WORKDIR /app
Removing intermediate container 586fe10936c0
 ---> 37c4cb868e8f
Step 24/28 : RUN pipenv install --dev
 ---> Running in c79e94ee6349
Creating a virtualenv for this project...
Pipfile: /app/Pipfile
Using /usr/local/bin/python3.6m (3.6.5) to create virtualenv...
Running virtualenv with interpreter /usr/local/bin/python3.6m
Using base prefix '/usr/local'
New python executable in /app/.venv/bin/python3.6m
Also creating executable in /app/.venv/bin/python
Installing setuptools, pip, wheel...done.

Virtualenv location: /app/.venv
Pipfile.lock not found, creating...
Locking [dev-packages] dependencies...

Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  You can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
  Hint: try $ pipenv lock --pre if it is a pre-release dependency.
Could not find a version that matches prompt-toolkit<2.0.0,<2.1.0,>=1.0.0,>=2.0.0
Tried: 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.8, 0.9, 0.10, 0.11, 0.12, 0.13, 0.14, 0.15, 0.16, 0.17, 0.18, 0.19, 0.20, 0.21, 0.22, 0.23, 0.25, 0.26, 0.28, 0.30, 0.31, 0.32, 0.32, 0.32, 0.33, 0.33, 0.33, 0.34, 0.34, 0.34, 0.35, 0.35, 0.35, 0.36, 0.36, 0.36, 0.37, 0.37, 0.37, 0.38, 0.38, 0.38, 0.39, 0.39, 0.39, 0.40, 0.40, 0.40, 0.41, 0.41, 0.41, 0.42, 0.42, 0.42, 0.43, 0.43, 0.43, 0.44, 0.44, 0.44, 0.45, 0.45, 0.45, 0.46, 0.46, 0.46, 0.47, 0.47, 0.47, 0.48, 0.48, 0.48, 0.49, 0.49, 0.49, 0.50, 0.50, 0.50, 0.51, 0.51, 0.51, 0.52, 0.52, 0.52, 0.53, 0.53, 0.53, 0.54, 0.54, 0.54, 0.55, 0.55, 0.55, 0.56, 0.56, 0.56, 0.57, 0.57, 0.57, 0.58, 0.58, 0.58, 0.59, 0.59, 0.59, 0.60, 0.60, 0.60, 1.0.0, 1.0.0, 1.0.0, 1.0.1, 1.0.1, 1.0.1, 1.0.2, 1.0.2, 1.0.2, 1.0.3, 1.0.3, 1.0.3, 1.0.4, 1.0.4, 1.0.4, 1.0.5, 1.0.5, 1.0.5, 1.0.6, 1.0.6, 1.0.6, 1.0.7, 1.0.7, 1.0.7, 1.0.8, 1.0.8, 1.0.8, 1.0.9, 1.0.9, 1.0.9, 1.0.10, 1.0.10, 1.0.10, 1.0.13, 1.0.13, 1.0.13, 1.0.14, 1.0.14, 1.0.14, 1.0.15, 1.0.15, 1.0.15, 2.0.1, 2.0.1, 2.0.1, 2.0.2, 2.0.2, 2.0.2, 2.0.3, 2.0.3, 2.0.3, 2.0.4, 2.0.4, 2.0.4
There are incompatible versions in the resolved dependencies.

The command '/bin/sh -c pipenv install --dev' returned a non-zero code: 1
```

**Comment by evgenyvoronov** [comment]
Please use python 3.6.7 for pipenv dependency.

**Comment by jredbeard** [comment]
Resolved in #219
